Implement secondary 'have' currency, used when 'want' equals primary 'have'

Irfy opened the pull request:
For example, if chaos is selected as the 'have' currency, when searching
for chaos price ('want'), a useless chaos:chaos listing is produced. With
this patch, a secondary 'have' currency can be defined, that is used when
'want' currency equals the primary 'have' currency.

Fixes #503

Eruyome commented:
Btw, adding options also requires to edit `\resources\ahk\userFolder\config_trade.ini`.

I'll wait a bit before adding this because it includes a config edit. This causes the config file to be overwritten (which annoys users) and some other changes that are planned or may be implemented soon also require config edits. So I'll bundle these changes into one release then.

Irfy commented:
![poe-trademacro-have2-setting](https://user-images.githubusercontent.com/736216/33804599-9a8cc62a-dda8-11e7-97af-4e7adcf54d82.png)
![poe-trademacro-have2-usage](https://user-images.githubusercontent.com/736216/33804623-1379a008-dda9-11e7-874f-17e241bbfc9b.png)

I have rebased my changes onto master and updated this pull request. Here's two screenshots just to give you a feel what my changes look like. I think the community could benefit from this.

Also, I accidentally closed the pull request by deleting the branch and cant repoen it any more.

Eruyome commented:
Finally added it, took me long enough^^

Irfy commented:
Thanks. I did freak out for a moment when I suddenly couldn't rebase my changes on top of master the other day :)
